libenkf/src/queue_config.c

```diff
@@ -68,6 +100,7 @@ job_queue_type * queue_config_alloc_job_queue(const queue_config_type * queue_co
 
   if (queue_config->max_submit_set)
     job_queue_set_max_submit(job_queue, queue_config->max_submit);
+
```

Comment:
Try to avoid making white-space changes in areas of the code without real changes.